rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -105,6 +99,24 @@ object Runtime {
       9L -> SystemProcesses.secp256k1Verify(space, dispatcher)
     )
 
+    lazy val replayDispatchTable: Map[Ref, Seq[ListChannelWithRandom] => Task[Unit]] = Map(
```

Comment:
these mappings are replicated 3 times:
1. in replayDispatchTable
2. in displatchTable
3. in procDefs

We should eliminate this duplication.

For replayDispatchTable and dispatchTable it's easy to extract a function which takes the space as parameter and returns the mapping. 

For procDefs we could do something like:

```val mappings = 
List(
("stdout", 1, 0L, SystemProcesses.stdout(_)),
("stdoutAck", 2, None, 1L, SystemProcesses.stdoutAck(_, dispatcher),
...)```

...and then build the two dispatch tables from this `mappings` datastructure